src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -58,6 +58,7 @@ class PhpDumper extends Dumper
     private $targetDirRegex;
     private $targetDirMaxMatches;
     private $docStar;
+    private $existingNames = array();
```

Comment:
No problem ;-)
Would you prefer ``$serviceIdToMethodNameMap`` [as proposed by @xabbuh](https://github.com/symfony/symfony/pull/18167#discussion-diff-56128507) ?